ztp/kube-compare-reference/compare.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+function read_dir() {
+for file in `ls $1`
```

Comment:
It's good practice to declare incoming variables with something like
```
  local dir=$1
```
instead of using $1 everywhere.

Likewise we should declare local variables like `file` as such as well:
```
  local file
```